.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,4 +5,4 @@ If this is your first PR to urllib3 please review the Contributing Guide:
 https://urllib3.readthedocs.io/en/latest/contributing.html
 
 Adhering to the Contributing Guide means we can review, merge, and release your change faster! :)
---->
+--->
```

Comment:
Can we reduce this to a file move, looks like a line ending got tweaked here.

Reply:
done !

Reply:
Let's revert the change on this file.